### PR TITLE
Drop support for Ruby 1.9.x

### PR DESCRIPTION
**Pull request by garethrees**
*tentative new proposal - drop 1.9 without updating the gems to sneak it into release 0.31 to tell a better story about dropping support for Wheezy and updating the Vagrant configs, split gem updates into a new ticket*

- [x] Remove from CI build
- [x] Remove references to `ruby1.9.1` package ~#4471~
- [x] Drop Wheezy support (EOL May 2018) #4462
- [x] Drop Precise support (EOL April 2017) https://github.com/mysociety/alaveteli/pull/4330 & https://github.com/mysociety/alaveteli/pull/4331
- [x] Note suggested workarounds (rbenv, BrightBox packages) #4460
- [x] Check `Gemfile` for gems we can now upgrade – ticket any complicated ones
  - https://github.com/mysociety/alaveteli/issues/4246

### CODE REVIEW

File: lib/quiet_opener.rb [hunk 1]
```diff
@@ -30,23 +27,17 @@ def quietly_try_to_open(url, timeout=60)
...
 
-# On Ruby versions before 2.0, we need to use the net-http-local gem
+# On Ruby versions before 2.0, we needed to use the net-http-local gem
 # to force the use of 127.0.0.1 as the local interface for the
 # connection.  However, at the time of writing this gem doesn't work
 # on Ruby 2.0 and it's not necessary with that Ruby version - one can
 # supply a :local_host option to Net::HTTP:start.  So, this helper
 # function is to abstract away that difference, and can be used as you
 # would Net::HTTP.start(host) when passed a block.
 def http_from_localhost(host)
```
Comment (garethrees) [inline lib/quiet_opener.rb]: I can't see this method even being called – could we remove it entirely?

Comment (lizconlan) [reply lib/quiet_opener.rb]: I couldn't find where it was being called from either ... I'll have a quick look online while waiting for specs to run (if neither of those turn up anything interesting, I'll remove it)

Comment (lizconlan) [reply lib/quiet_opener.rb]: (Got a clean spec run, no clues online so rebasing to remove that method before merge)

**Comment by lizconlan** [comment]
Merged into the release branch

**Pull request closed without merging**